src/mem/mem_ctrl.hh

```diff
@@ -269,6 +276,40 @@ class MemCtrl : public QoS::MemCtrl
      */
     MemoryPort port;
 
+    // Add a tag store map
+    // TODO: look at what the intialization
+    // values should be
+    class tag_entry
+    {
+    public:
+      Addr tag = -1;
```

Comment:
Addr of -1 doesn't make sense (it's an unsigned). I suggest using `0` and adding a `valid` field